protoc-rust/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "protoc-rust"
 version = "3.0.0"
 authors = ["Stepan Koltsov <stepan.koltsov@gmail.com>"]
-license = "MIT/Apache-2.0"
+license = "MIT/"
```

Comment:
Looks like wrong slash here

Reply:
ups, yeah the slash should not be there

Reply:
> A question: do we need `LICENSE.txt` in every directory? This is because these files are packed into Cargo package?

It is good style. For once it is distributed in source, and the license file should be there. Also the cargo API allows to retrieve the license file from the package, but this only works, if it is distributed with it. Some people and companies use automated toolchains, to verify their licenses, in order to ensure they can not be sued. Such a tool might throw an error if it can not find the license together with the source.

Reply:
So, ... , fixed the slash